src/bundle.scss

```diff
@@ -0,0 +1,42 @@
+@forward "./normalize";
```

Comment:
You have to hide all the development files from SCSS compiler, so you call watch command without specifying the source and target, and the compiler doesn't watch and compile everything